README.md

```diff
@@ -79,6 +79,7 @@ type options = {
 	password?: string, // The password for the username you're connecting to. Leave this blank if the account is unregistered.
 	server?: string, // The server to which you wish to connect to - defaults to 'sim3.psim.us'.
 	port?: number, // The port on which you're connecting to. Can also be specified in server as `url:port`, in which case leave this field blank.
+	serverProtocol?: number, // The protocol used for the websocket connection. Defaults to wss, but can be changed to ws (insecure).
```

Comment:
`'wss' | 'ws' | string` (or just `string`)

Reply:
... I really need to sleep lol. Apologies